oak_remote_attestation/proto/v1/messages.proto

```diff
@@ -111,3 +111,13 @@ message InvokeResponse {
   // TODO(#3843): Use explicit `oak_crypto` messages.
   bytes encrypted_body = 1;
 }
+
+message GetEncryptionKeyRequest {
+  // Encrypted attestation evidence of the dependent enclave.
+  bytes encrypted_attestation_bundle = 1;
```

Comment:
The attestation evidence (bundle) is public, so you can just use the corresponding proto message.

Also I'm currently trying to rename the [`AttestationBundle`](https://github.com/project-oak/oak/blob/c7027badbd0d6dbb2256f25fa21740b31cc43781/oak_remote_attestation/proto/v1/messages.proto#L46-L54) to [`EndorsedEvidence`](https://github.com/project-oak/oak/blob/c7027badbd0d6dbb2256f25fa21740b31cc43781/oak_remote_attestation/proto/v1/endorsement.proto#L59-L67), since bundle was not a correct term.

Reply:
Noted. I'll use ```AttestationBundle``` for this pull request unless you have your changes already in main.